src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/furtherevidence/actionfurtherevidence/ActionFurtherEvidenceAboutToSubmitHandlerTest.java

```diff
@@ -195,6 +198,58 @@ public void givenAValidPostponementRequest_thenSscsDocumentTypeIsPostponementReq
                 .anyMatch(note -> note.getValue().getNoteDetail().equals("Request Detail Test")), is(true));
     }
 
+    private void setUpCaseDataForSetAside() {
```

Comment:
I have been strongly advised against this kind of setup by Lucas. Reason being the tests become too interlinked in an unclear way. ie If something changes about one of them, you would have to refactor or somehow alter this function, which would then affect the other tests. Lucas emphasized that tests should be self-contained and -evident of what they are setting up and testing. (Perhaps also why unnecessary stubbings exception exists for Mockito)

His preferred options are:
1) Inline the code so that it is repeated in each test - this makes it very clear what is the setup when looking at each test on its own. Not a huge crime for a small number of tests, as here.
2) Abstract this test class and place these tests in their own extends class. This allows you to make a new BeforeEach which only applies to these tests.

Up to you which you do. 1 would be fine here imo, but you may want to consider 2 if there are more tests expected further down the line.

Reply:
I think Option 1 is better here, I'd rather we don't abstract to new classes as that could get a bit messy (similar to having multiple setup methods in a single class). I'll make the tests more generic to fit the other post hearing types however to save on further overhead (did a quick Jira scan and the case states / setups are the same for corrections, liberty to apply et al, so it makes sense).